Index: 05_lesson/lesson5_task_01.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/05_lesson/lesson5_task_01.py b/05_lesson/lesson5_task_01.py
new file mode 100644
--- /dev/null	(date 1752430829460)
+++ b/05_lesson/lesson5_task_01.py	(date 1752430829460)
@@ -0,0 +1,9 @@
+from time import sleep
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+driver = webdriver.Chrome()
+driver.get("http://uitestingplayground.com/classattr")
+driver.find_element(By.XPATH,"//button[contains(concat(' ', normalize-space(@class), ' '), ' btn-primary ')]").click()
+sleep(10)
\ No newline at end of file
